config/tls.go

```diff
@@ -57,6 +61,9 @@ func (t *TLS) MakeConfig(serverName string) (*tls.Config, error) {
 		return nil, nil
 	}
 
+	isRawPem := func(s string) bool { return strings.HasPrefix(strings.TrimSpace(s), "-----BEGIN") }
```

Comment:
Maybe just test if it can be parsed as PEM instead?

PEM files can contain additional data outside these blocks can for example be used to add a comment above each certificate in a file containing multiple certificates.

Should be as simple as that (using [`encoding/pem.Decode()`](https://pkg.go.dev/encoding/pem#Decode)):

```go
block, _ := pem.Decode(s)
return block != nil
```